cfpb/cfgov-refresh
# HMDA: migrate dynamic content cf-icon to SVG

## Changes

- Convert dynamically added cf-icon alert to SVG icon.
- Adjust top margin on file size notice.

## Testing

1. Enable the `HMDA_LEGACY_PUBLISH` flag in local wagtail.
2. Visit http://localhost:8000/data-research/hmda/explore and enter "2017" and "2012" for the "Select year(s) of data" and choose "All records" for "Select suggested filters". Expand the location to see the error notification and ensure there's an icon in the notification.
3. Also, see that file size warning text and popup button line up at the bottom of the page.

## Screenshots

Before:
<img width="819" alt="Screen Shot 2019-05-15 at 11 16 24 AM" src="https://user-images.githubusercontent.com/704760/57787528-8fa3e180-7703-11e9-8d37-dcc0d4683294.png">

<img width="220" alt="Screen Shot 2019-05-15 at 11 16 14 AM" src="https://user-images.githubusercontent.com/704760/57787521-8a469700-7703-11e9-9ecb-3c3aff27bd7c.png">

After:
<img width="836" alt="Screen Shot 2019-05-15 at 11 17 47 AM" src="https://user-images.githubusercontent.com/704760/57787551-99c5e000-7703-11e9-99f0-b013dc05dd59.png">

<img width="210" alt="Screen Shot 2019-05-15 at 11 26 28 AM" src="https://user-images.githubusercontent.com/704760/57787981-515af200-7704-11e9-870b-7acfca942e9c.png">
